### PR TITLE
added: script to sync drive-db files

### PR DESCRIPTION
**Pull request by yashre-bh**

### CODE REVIEW

File: sync.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    if(fileExists):
+        print(file['title'] + ' file already in database...')
+            
+    else:
+        addFileInDB(file)
+        filesSynced = filesSynced + 1
+        
+            
+def courseScan(course, departments, depts):
+    service = driveinit()
```
Comment (ayanchoudhary) [inline sync.py]: no need to init this service multiple times, just start it once at the beginning of the flow and pass the service to the functions

File: sync.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    for folder in departments[depts][course]:
+        if course != 'id':
+            if folder != 'id':
+                pickFolder = departments[depts][course][folder]
```
Comment (ayanchoudhary) [inline sync.py:111]: I don't see us separating the `_review` folders out, we should ignore the files existing in those folders

File: sync.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+def printContents():
+    try:
+        qry = '''
+            SELECT * FROM resources_file 
+            '''
+        cursor.execute(qry)
+        records = cursor.fetchall()
+
+        for row in records:
+            print (row[0],"\t",row[1],"\t",row[2],"\t",row[3],"\t",row[4],"\t",row[5],"\t",row[6],"\t",row[7],"\t",row[8],"\t",row[9])
```
Comment (ayanchoudhary) [inline sync.py]: give the names of the fields as well, will make the print look meaningful

File: sync.py [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+import datetime
+
+
+today = datetime.datetime.today()
+date_today = today.strftime("%Y-%m-%d")
+filesSynced = 0
+
+
+try:
+
```
Comment (ayanchoudhary) [inline sync.py]: remove blank line

File: sync.py [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    connection = psycopg2.connect(database="studyportal", 
+                            user = "studyportal", 
+                            password = "studyportal", 
+                            host = "localhost", 
+                            port = "5432")
```
Comment (ayanchoudhary) [inline sync.py:19]: did you try connecting this with the postgres running in the docker container?

**Review by ayanchoudhary — CHANGES_REQUESTED** [review]

File: sync.py (hunk 5 above)
Comment (yashre-bh) [reply sync.py:19]: yes, got connected with postgres in docker only

File: sync.py (hunk 1 above)
Comment (yashre-bh) [reply sync.py]: iska issue, check slack once please :)

**Review by ayanchoudhary — APPROVED** [review]
Looks good and the PR works as expected

**Pull request merged**